docs/cluster_howto/exercise_R/exercise.rst

```diff
@@ -64,25 +60,30 @@ In this task, we use the script ``magic_cal.R``. This script uses the ``magic``
 
    .. code-block:: bash
 
-        $ echo "Rscript $PWD/magic_cal.R 5" | qsub -N "magic_cal" -l walltime=00:10:00,mem=256mb
-        11082769.dccn-l029.dccn.nl
+        $ sbatch --time=00:10:00 --mem=256mb --job-name=magic_R << EOF
+        #!/bin/bash
+        Rscript magic_cal.R 5
+        EOF
 
 #. wait the job to finish, and check the output of the job. Do you get same results as running interactively?
 
-#. run five batch jobs in parallel to run the ``magic_cal.R`` with matrices in dimention 5,6,7,8,9.
+#. run five batch jobs in parallel to run the ``magic_cal.R`` with matrices in dimension 5,6,7,8,9.
 
    .. code-block:: bash
 
-        $ for d in {5 .. 9}; do
-            echo "Rscript $PWD/magic_cal.R $d" | qsub -N "magic_cal_$d" -l walltime=00:10:00,mem=256mb;
+        $ for d in {5..9}; do
+        sbatch --time=00:10:00 --mem=256mb --job-name="magic_cal_$d" << EOF
+        #!/bin/bash
+        Rscript magic_cal.R $d
+        EOF
```

Comment:
@schoffelen this can also be simplified by using the `--wrap` option of `sbatch`, e.g.

```bash
sbatch --time=00:10:00 --mem=256mb --job-name="magic_cal_$d" --wrap="Rscript magic_cal.R $d"
```